contents/handbook/engineering/feature-pricing.md

```diff
@@ -29,6 +29,8 @@ It's important that every time we create a new feature or shift our pricing plan
 
 Sometimes these principles still leave room for questioning - what, if any, should be available in the free tier? What about enterprise customers? For these types of questions, we've defined a runbook for deciding which plans and at what limits features should generally be assigned to.
 
+- In general we should match the pricing of the cheapest big competitor for that product to make it a no brainer to use PostHog
+  - We can do this because we can upsell customers multiple of our other products. The total ACV higher, even if the per-product ACV is lower. It's better for customers as they get all these tools that are well integrated for the cheapest possible price.
```

Comment:
```suggestion
- In general, we should match the pricing of the cheapest big competitor for that product, so long as the unit economics make sense, to make it a no-brainer to use PostHog.
  - We can do this because we can upsell customers multiple of our other products. The total ACV is higher even if the per-product ACV is lower. It's better for customers as they get all these tools that are well integrated for the cheapest possible price.
```